store/tikv/tests/2pc_test.go

```diff
@@ -747,11 +755,17 @@ func (s *testCommitterSuite) TestDeleteYourWriteCauseGhostPrimary(c *C) {
 	txn1 := s.begin(c)
 	txn1.DelOption(kv.Pessimistic)
 	s.store.ClearTxnLatches()
-	txn1.Get(context.Background(), k1)
-	txn1.GetMemBuffer().SetWithFlags(k1, []byte{0}, kv.SetPresumeKeyNotExists)
-	txn1.Set(k2, []byte{1})
-	txn1.Set(k3, []byte{2})
-	txn1.Delete(k1)
+	_, err := txn1.Get(context.Background(), k1)
+	// to fix: key not exist
```

Comment:
Could we remove this comment? It seems that `k1` should not exist.

Reply:
Ok